From 07e904fbde2cbffb7fd757ce7d6151ecdfa3ab04 Mon Sep 17 00:00:00 2001
From: Stanislaw Gruszka <sgruszka@redhat.com>
Date: Fri, 5 Sep 2014 11:23:47 -0400
Subject: [redhat] configs: enable CONFIG_ATH9K_WOW

Message-id: <1409916232-5882-11-git-send-email-sgruszka@redhat.com>
Patchwork-id: 91439
O-Subject: [RHEL7 PATCH 10/15] configs: enable CONFIG_ATH9K_WOW
Bugzilla: 1136324
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: John Greene <jogreene@redhat.com>

BZ#1136324

Decription:
WoWLAN support became optional on updated driver. Enable that feature
since customers utilize it.

Brew:
https://brewweb.devel.redhat.com/taskinfo?taskID=7918052

Testing:
Not tested.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_ATH9K_WOW b/redhat/configs/generic/CONFIG_ATH9K_WOW
new file mode 100644
index 0000000..169bfaa
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_ATH9K_WOW
@@ -0,0 +1 @@
+CONFIG_ATH9K_WOW=y
-- 
1.7.1